wallet-new/src/Cardano/Wallet/Kernel/Transactions.hs

```diff
@@ -208,9 +232,10 @@ estimateFees activeWallet@ActiveWallet{..} genChangeAddr signAddress options acc
     case res of
          Left e  -> return . Left . EstFeesTxCreationFailed $ e
          Right tx -> -- calculate the fee as the difference between inputs and outputs.
-             -- NOTE(adn) Apparently we shouldn't worry about the 'ExpenseRegulation'
-             -- affecting the way we sum, as no matter who pays for the fee,
-             -- the final difference won't be affected.
+             -- NOTE(adn) In case of 'SenderPaysFee' is practice there might be a slightly
+             -- increase of the projected fee in the case we are forced to pick "yet another input"
+             -- to be able to pay the fee, which would, in turn, also increase the fee due to
+             -- the extra input being picked.
```

Comment:
It is also important to note that _if_ this happens, it may in fact be the case that we give you an estimated fee where in reality we would _fail_ to produce a transaction at all.